src/CommerceModel.BetterRetail/artifacts/OOE/BetterRetail/ImportEmailTemplatesActivity/Global/OrderReturnCreated/Contents/Html/en-CA.html

```diff
@@ -162,6 +169,11 @@ <h1 style="font-size: 32px;">Hi {{Return.CustomerFirstName}} {{Return.CustomerLa
                                                     <strong>{{ Return.TotalTax | money:"en-CA" }}</strong>
                                                 </td>
                                             </tr>
+                                            <tr>
+                                                <td colspan="3">&nbsp;</td>
+                                                <td style="line-height: 20px; text-align: left;" colspan="2">Total Adjustment:</td>
+                                                <td style="line-height: 20px; padding: 5px 10px 5px 0px;" colspan="1"><strong>{{ Return.TotalAdjustment | money:"en-CA" }}</strong></td>
+                                            </tr>
```

Comment:
1) Display only when TotalAdjustment !== 0  (should be absent when it is not present)
2) EN: "Adjustment Amount:"
   FR: "Montant de l'ajustement:"